app/controllers/procurement_area_memberships_controller.rb

```diff
@@ -35,13 +35,11 @@ def destroy
   private
 
   def membership_params
-    { uid: params.delete(:membership_uid), type: params.delete(:membership_type) }
+    { id: params.delete(:membership_id) }
   end
 
   def organisations
-    all_organisations_by(type: %w(law_firm law_office court custody_suite)).map do |organisation|
-      OrganisationPresenter.new(organisation)
-    end
+    Organisation.where(organisation_type: %w(law_firm law_office court custody_suite))
```

Comment:
So here I suggest to either remove the types as @ansonK suggested, because we don't need them in this app. All organisations are law firms. In fact, we could even rename the model. 

If we keep them, we should possibly move this to a scope method.

And in general, should `custody_suite` and `court` be allowed here in the first place?